### PR TITLE
Adding missing code in FormRecordListActivity (incomplete forms search)

### PR DESCRIPTION
**Pull request by dcluna**
Also centering the logo in connection_diagnostic screen.

### CODE REVIEW

**Comment by dcluna** [comment]
Better to isolate the action bar logic in CommCareActivity itself (the same logic is used in other class), to avoid code duplication

File: app/src/org/commcare/dalvik/activities/FormRecordListActivity.java [hunk 1]
```diff
@@ -93,7 +96,8 @@
     private ImageButton barcodeButton;
     private Spinner filterSelect;
     private ListView listView;
-    
+    private SearchView searchView;
```
Comment (phillipm) [inline app/src/org/commcare/dalvik/activities/FormRecordListActivity.java]: The linter says that this variable can be made local to the onCreateOptionsMenu method.

**Comment by phillipm** [comment]
Looks good, I'll merge this after you make "searchView" a method-level variable instead of a class-level variable.

File: app/src/org/commcare/dalvik/activities/FormRecordListActivity.java (hunk 1 above)
Comment (dcluna) [inline app/src/org/commcare/dalvik/activities/FormRecordListActivity.java]: After the last commit, this linter warning was removed

**Comment by phillipm** [comment]
Thanks for addressing that and nice unification of repeated code!
I tidied up a few style things and merged.

**Pull request merged**